doc/related_projects.rst

```diff
@@ -57,6 +60,8 @@ enhance the functionality of scikit-learn's estimators.
 - `mlxtend <https://github.com/rasbt/mlxtend>`_ Includes model visualization
   utilities.
 
+- `Fast svmlight / libsvm file loader <https://github.com/mblondel/svmlight-loader>`_
```

Comment:
I'm not certain whether this is current, i.e. still much faster than what's in `sklearn.datasets`. @mblondel?